shap/__init__.py

```diff
@@ -16,6 +16,7 @@
 from .explainers._kernel import Kernel as KernelExplainer
 from .explainers._sampling import Sampling as SamplingExplainer
 from .explainers._tree import Tree as TreeExplainer
+from .explainers._gpu_tree import GPUTree as GPUTreeExplainer
```

Comment:
I think this is the right choice, but my goal is to point users at explainers.GPUTree eventually.